server/middleware/link/setupRequestLink.ts

```diff
@@ -1,27 +1,13 @@
 import express, { Router } from 'express'
-import session from 'express-session'
 import flash from 'connect-flash'
 import RequestLinkController from '../../routes/link/RequestLinkController'
-import config from '../../config'
 import MagicLinkService from '../../services/link/MagicLinkService'
 
 export default function setUpRequestLink(magicLinkService: MagicLinkService): Router {
   const router = express.Router()
   const requestLinksController = new RequestLinkController(magicLinkService)
 
-  // TODO This uses the same max session age and secret as for a logged in user. Consider separating these into different env vars.
-  router.use(
-    session({
-      cookie: { maxAge: config.session.expiryMinutes * 60000 },
-      store: new session.MemoryStore(),
-      saveUninitialized: true,
-      resave: true,
-      secret: config.session.secret,
-    })
-  )
   router.use(flash())
-  router.use(express.json())
```

Comment:
Removed these lines as they are already setup in `setupRequestParsing.ts`. Using flash isnt setup though, so I've left that for the time being, but it doesnt feel like something we want/need to setup for each controller/path setup?

Reply:
I had to add both in the PoC to get it working - but that may be because of misconfiguration in other areas.